### PR TITLE
TASK: _*_SCENARIO

### PR DESCRIPTION
**Pull request by slashkeyvalue**

### CODE REVIEW

**Comment by alloc8or** [comment]
It's actually scenario _point_, not scenario.
So please replace every occurence of ``SCENARIO`` with ``SCENARIO_POINT``.
Other than that it looks good 👍

File: natives.json [hunk 1]
```diff
@@ -95365,15 +95365,15 @@
 			"build": "1207"
 		},
 		"0xEEE4829304F93EEE": {
-			"name": "_0xEEE4829304F93EEE",
+			"name": "_DELETE_SCENARIO",
```
Comment (alloc8or) [inline natives.json:95368]: Just realized that this definitely begins with ``SET_SCENARIO_POINT_``.
So I'm guessing it's either _ACTIVE, _ENABLED or _DISABLED.
Could you verify what it does exactly and name it accordingly? (``_SET_SCENARIO_POINT_*``)

Comment (slashkeyvalue) [reply natives.json:95368]: Sure, i will confirm it and update everything as soon as i get a result

**Comment by slashkeyvalue** [comment]
I did rename `_GET_CLOSEST_SCENARIO_OF_TYPE` to `_GET_CLOSEST_ACTIVE_SCENARIO_POINT_OF_TYPE` as it doesn't return a scenario if i have deactivated it with `_SET_SCENARIO_POINT_ACTIVE`

**Comment by alloc8or** [comment]
One last thing that I'd like to have changed is ``_GET_CLOSEST_ACTIVE_SCENARIO_POINT_OF_TYPE`` -> ``_FIND_CLOSEST_ACTIVE_SCENARIO_POINT_OF_TYPE`` (alphabetical position indicates it starts with 'F').

**Pull request merged**